code/modules/food_and_drinks/food/snacks_cake.dm

```diff
@@ -175,10 +175,11 @@
 
 /obj/item/reagent_containers/food/snacks/store/cake/birthday
 	name = "birthday cake"
-	desc = "Happy Birthday little clown..."
+	desc = "Happy Birthday! You get an awesome gift if you eat this whole!"
 	icon_state = "birthdaycake"
 	slice_path = /obj/item/reagent_containers/food/snacks/cakeslice/birthday
 	slices_num = 5
+	trash = /obj/item/clothing/head/hardhat/cakehat //super sekrit club
```

Comment:
Replace this with crafting/cooking, it makes no sense that you eat a whole cake and then a cake hat appears out of nowhere.

Reply:
A little nonsense has never hurt this codebase, alright fine, it's now on microwave act.

Reply:
Forcing you to eat the cake gates this behind getting fat/having to forcefeed the cake to someone though. While nonsensical, it makes it harder to do than just plain crafting.